### PR TITLE
feat(kumactl) add install dns command

### PR DESCRIPTION
**Pull request by nickolaev**
### Summary

This is a complement to #798 which adds a convenient `kumactl install dns` command for Kubernetes deployment, where setting up dns is essentially to edit the relevant CoreDNS ConfigMap.
Natively this can be done by editing it like this: `kubectl edit -n kube-system cm/coredns`. The proposed command will read the `cm/coredns` add the relevant kuma DNS entry and will print the resulting YAML to stdout. The practical usage is: `kumactl install dns | kubectl apply -f -`

The command requires the KUBECONFIG env variable to be set and point to the K8s cluster which will be configured.
Testing shall be done as part of the DNS PR or follow up.

https://github.com/coredns/deployment/blob/master/kubernetes/CoreDNS-k8s_version.md

### Documentation

- [ ] TBD

### CODE REVIEW

File: app/kumactl/cmd/install/install_dns.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+const appendTemplate = `kuma:53 {
+        errors
+        cache 30
+        %s . %s:5653
+    }`
+
+var resourceHeader = []byte("apiVersion: v1\nkind: ConfigMap\n")
+
+func newInstallDNS() *cobra.Command {
```
Comment (lobkovilya) [inline app/kumactl/cmd/install/install_dns.go:25]: We have tests for every `kumactl install ...` command. Could you please add one for this

File: app/kumactl/cmd/install/install_dns.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+				return errors.Wrap(err, "Failed to output the generated resources")
+			}
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func getCoreFileForwardVerb(corefile string) string {
```
Comment (lobkovilya) [inline app/kumactl/cmd/install/install_dns.go:86]: Are these words "proxy" or "forward" mandatory? Is it possible that corefile doesn't contain any of them

File: app/kumactl/cmd/install/install_dns.go (hunk 1 above)
Comment (nickolaev) [reply app/kumactl/cmd/install/install_dns.go:25]: The test will be part of the E2E, you can not really test it without a valid K8s controller. There is a sentence about this at the end of the PR description.

File: app/kumactl/cmd/install/install_dns.go (hunk 2 above)
Comment (nickolaev) [reply app/kumactl/cmd/install/install_dns.go:86]: They are not mandatory, but is the typical usage of CoreDNS in K8s. Of course, anyone can edit their CoreDNS Corefile, but in such case, they would not need our kumactl install DNS helper, they already know their way through this.

Comment (jakubdyszkiewicz) [inline app/kumactl/cmd/install/install_dns.go:86]: Maybe some comment here why do we do this forward verb. That it depends on the CoreDNS version

**Pull request merged**